### PR TITLE
Migrated from cluster_token resource to EKS cluster attribute

### PR DESCRIPTION
**Pull request by andrejatcastai**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -94,7 +94,7 @@ resource "helm_release" "castai_agent" {
 
   set_sensitive {
     name  = "apiKey"
-    value = castai_cluster_token.cluster_token.cluster_token
```
Comment (aldor007) [inline main.tf:97]: IMO you should also update provider version requirements https://github.com/castai/terraform-castai-eks-cluster/blob/main/versions.tf#L12

Comment (andrejatcastai) [reply main.tf:97]: Yes, you are right, fixed that. Now dependency on the provider defined as ">= 2.0.0"

**Comment by aldor007** [comment]
@andrejatcastai you have a conflict in this PR

File: versions.tf [hunk 2]
```diff
@@ -9,7 +9,7 @@ terraform {
     }
     castai = {
       source  = "castai/castai"
-      version = ">= 0.25.0"
+      version = ">= 2.0.0"
```
Comment (varnastadeus) [inline versions.tf:12]: introduce `cluster_token` in non braking release, then version will be 1.x.x

**Pull request closed without merging**